spec/lib/dor/text_extraction/speech_to_text_spec.rb

```diff
@@ -45,7 +45,7 @@
 
       context 'when the object has no files that can be STTed' do
         let(:first_fileset_structural) { instance_double(Cocina::Models::FileSetStructural, contains: [text_file]) }
-        let(:second_fileset_structural) { instance_double(Cocina::Models::FileSetStructural, contains: [text_file, text_file]) }
+        let(:second_fileset_structural) { instance_double(Cocina::Models::FileSetStructural, contains: [text_file, text_file2]) }
```

Comment:
small typo from a previous PR, turned out to have no impact on the test, but just fixing because